kolibri/plugins/facility/assets/src/views/FacilityTopNav.vue

```diff
@@ -21,28 +21,30 @@
     mixins: [commonCoreStrings],
     computed: {
       links() {
+        const facility_id = this.$store.getters.activeFacilityId; // Ensure we stay in the current facility
```

Comment:
One other possibility here would be to read the route parameter, which would mean navigation within the default facility would not inject the route parameter, but other facilities would.

Reply:
Great call - will update tomorrow thanks!